src/_common/community/card-base/card-base-inline.styl

```diff
@@ -5,6 +5,20 @@ $-thumbnail-size = 80px
 $-thumbnail-size-sm = 48px
 $-bg-color-base = var(--theme-bg)
 
+.-unread-blip
+	position: absolute
+	top: 16px
+	left: 24px
+	height: 12px
+	width: 12px
+	border: $border-width-large solid var(--theme-bg)
+	background-color: var(--theme-fg)
```

Comment:
I think we should use theme-notice for this one. Thats the color of the blips on the channel images as well.

Reply:
I decided against that since it's not an actual "notification" it's just "unread"